Documentation/ApiOverview/FormProtection/Index.rst

```diff
@@ -1,150 +1,167 @@
+different UIDs) from the same table, those values should be different.
 
-For editing a tt_content record, the call could look like this:
+For editing a :sql:`tt_content` record, the call could look like this:
 
-.. code-block:: php
+..  code-block:: php
 
-   $formToken = TYPO3\CMS\Core\FormProtection\FormProtectionFactory::get()->generateToken('tt_content', 'edit', $uid);
+    $formToken = \TYPO3\CMS\Core\FormProtection\FormProtectionFactory::get()
+        ->generateToken('tt_content', 'edit', $uid);
 
-At the end of the form, you need to persist the tokens. This makes sure that generated tokens get saved,
-and also that removed tokens stay removed:
+Finally, you need to persist the tokens. This makes sure that
+generated tokens get saved, and also that removed tokens stay removed:
 
-.. code-block:: php
+..  code-block:: php
 
-   TYPO3\CMS\Core\FormProtection\FormProtectionFactory::get()->persistTokens();
+    \TYPO3\CMS\Core\FormProtection\FormProtectionFactory::get()
+        ->persistTokens();
 
-In BE lists, it might be necessary to generate hundreds of tokens.
-So the tokens do not get automatically persisted after creation for performance reasons.
+In backend lists, it might be necessary to generate hundreds of tokens.
+So, the tokens are not automatically persisted after creation for performance
+reasons.
 
 When processing the data that has been submitted by the form,
 you can check that the form token is valid like this:
 
-.. code-block:: php
+..  code-block:: php
+
+    if ($dataHasBeenSubmitted &&
+        \TYPO3\CMS\Core\FormProtection\FormProtectionFactory::get()->validateToken(
+            (string) \TYPO3\CMS\Core\Utility\GeneralUtility::_POST('formToken'),
+            'BE user setup',
+            'edit'
+        ) ) {
+        // process the data
+    } else {
+        // No need to do anything here, as the backend form protection will
+        // create a flash message for an invalid token
+    }
 
-   if ($dataHasBeenSubmitted &&
-      TYPO3\CMS\Core\FormProtection\FormProtectionFactory::get()->validateToken(
-         (string) \TYPO3\CMS\Core\Utility\GeneralUtility::_POST('formToken'),
-         'BE user setup', 'edit'
-      ) ) {
-      // processes the data
-   } else {
-      // no need to do anything here as the BE form protection will create a
-      // flash message for an invalid token
- }
 
-Note that :code:`validateToken` invalidates the token with the token ID.
-So calling the validation with the same parameters twice in a row
-will always return :code:`FALSE` for the second call.
+..  note::
+    The :php:`validateToken()` method invalidates the token with the token ID.
+    So calling the validation with the same parameters twice in a row will
+    always return :php:`false` for the second call.
 
-It is important that the tokens get validated **before** the tokens are persisted.
-This makes sure that the tokens that get invalidated by :code:`validateToken`
-cannot be used again.
+..  important::
```

Comment:
do not use "important" as it is displayed green like a note. Use .. attention instead